src/frontend/screens/Settings/sections/GamesSettings/index.tsx

```diff
@@ -82,101 +118,75 @@ export default function GamesSettings({ useDetails = true }: Props) {
         </p>
       )}
 
-      {!nativeGame && (
-        <>
-          <Collapsible
-            isOpen
-            isCollapsible={useDetails}
-            summary={isLinux ? 'Wine' : 'Wine/Crossover'}
-          >
+      <Tabs value={value} onChange={handleChange} aria-label="settings tabs">
+        {!isWin && <Tab label="Wine" />}
+        <Tab label={t('settings.navbar.other', 'Other')} />
+        {!isCrossover && !isWin && (
+          <Tab label={t('settings.navbar.advanced', 'Advanced')} />
+        )}
+        {hasCloudSaves && (
+          <Tab label={t('settings.navbar.sync', 'Cloud Saves Sync')} />
+        )}
+      </Tabs>
+
+      <TabPanel value={value} index={0}>
+        {!nativeGame && (
+          <>
             <WineVersionSelector />
             <WinePrefix />
             <CrossoverBottle />
-
             {!isCrossover && (
               <>
                 <AutoDXVK />
                 {isLinux && (
                   <>
                     <AutoDXVKNVAPI />
-
                     <AutoVKD3D />
-
-                    <EacRuntime />
-
-                    <BattlEyeRuntime />
                   </>
                 )}
-                <Tools />
-              </>
-            )}
-          </Collapsible>
-        </>
-      )}
-
-      <Collapsible
-        isOpen={nativeGame}
-        isCollapsible={useDetails}
-        summary={
-          nativeGame
-            ? t('settings.navbar.advanced', 'Advanced')
-            : t('settings.navbar.other', 'Other')
-        }
-      >
-        <AlternativeExe />
-
-        <ShowFPS />
-
-        {!nativeGame && <EnableDXVKFpsLimit />}
-
-        {!isWin && !nativeGame && (
-          <>
-            <EnableEsync />
-
-            {isLinux && (
-              <>
+                <EnableEsync />
                 <EnableFsync />
-
-                <PreferSystemLibs />
-
                 <EnableFSR />
-
-                <GameMode />
+                <Tools />
               </>
             )}
           </>
         )}
+      </TabPanel>
 
-        <UseDGPU />
-
-        {isLinux && <Mangohud />}
-
-        <SteamRuntime />
-
-        <IgnoreGameUpdates />
-
-        <OfflineMode />
-
-        <EnvVariablesTable />
-
-        <WrappersTable />
+      <TabPanel value={value} index={1}>
+        {!isCrossover && (
+          <>
+            <GameMode />
+            <ShowFPS />
+            <Mangohud />
+            <BattlEyeRuntime />
+            <EacRuntime />
```

Comment:
this is showing on Mac but I understand EAC doesn't work on Mac (I haven't checked yet if it's an issue also on main though)

Reply:
both runtimes are linux exclusive

Reply:
fixed